src/System.Private.CoreLib/shared/System/Text/UTF8Encoding.cs

```diff
@@ -155,7 +156,7 @@ public override unsafe int GetByteCount(char[] chars, int index, int count)
         public override unsafe int GetByteCount(string chars)
         {
             // Validate input
-            if (chars==null)
+            if (chars == null)
                 throw new ArgumentNullException("s");
```

Comment:
Separate from your PR, why is this `"s"` rather than `nameof(chars)`?
cc: @tarekgh

Reply:
This is just copied from the old code from the full framework, [http://index/?query=Utf8Encoding.GetByte&rightProject=mscorlib&file=system%5Ctext%5Cutf8encoding.cs&line=164](http://index/?query=Utf8Encoding.GetByte&rightProject=mscorlib&file=system%255Ctext%255Cutf8encoding.cs&line=164)

yes, we should use nameof(chars)

---
In reply to: [272208714](https://github.com/dotnet/coreclr/pull/23708#discussion_r272208714) [](ancestors = 272208714)